app/javascript/src/components/Dashboard/Notes/Card.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "@bigbinary/neeto-icons";
+import * as dayjs from "dayjs";
+import { Dropdown, Typography, Tag, Avatar, Tooltip } from "neetoui/v2";
+
+const Card = ({ note, setNoteIdToDelete }) => {
```

Comment:
I forgot to destructor `note`.  I'll resolve it after completing the pending issues.